ci/test_cpp_memcheck.sh

```diff
@@ -9,7 +9,6 @@ source ./ci/test_cpp_common.sh
 rapids-logger "Memcheck gtests with rmm_mode=cuda"
 
 ./ci/run_cudf_memcheck_ctests.sh \
-    --gtest_output=xml:"${RAPIDS_TESTS_DIR}${test_name}.xml" \
  && EXITCODE=$? || EXITCODE=$?;
```

Comment:
Can we intend this line too? Otherwise the continuation is lost. Alternatively, maybe just merge lines 11 and 12.